client/i18n/locales/english/intro.json

```diff
@@ -1932,30 +1932,17 @@
       "phko": { "title": "148", "intro": [] },
       "wjzt": { "title": "149", "intro": [] },
       "tsdq": { "title": "150", "intro": [] },
-      "workshop-calculator": {
-        "title": "Build a Calculator",
+      "kkix": { "title": "151", "intro": [] },
```

Comment:
This does not look quite right to me. If I'm not wrong we should have the following:
```
      "wjzt": { "title": "149", "intro": [] },
      "tsdq": { "title": "150", "intro": [] },
      "workshop-calculator": {
        "title": "Build a Calculator",
        "intro": [
          "In this workshop, you will review working with functions by building a calculator."
        ]
      },
      "lab-email-masker": {
        "title": "Build an Email Masker",
        "intro": [
          "In this lab, you'll build an email masker that will take an email address and obscure it."
        ]
      },
      "lab-sentence-maker": {
        "title": "Build a Sentence Maker",
        "intro": [
          "In this lab, you will create different stories by assigning different words to different variables."
        ]
      },
      "workshop-loan-qualification-checker": {
        "title": "Build a Loan Qualification Checker",
        "intro": [
          "In this workshop, you will continue to learn how to work with conditionals by building a loan qualification checker app.",
          "You will learn more about <code>if</code> statements, and how to use comparison operators and multiple conditions in an <code>if</code> statement."
        ]
      },
      "lab-leap-year-calculator": {
        "title": "Build a Leap Year Calculator ",
        "intro": [
          "In this lab you will use conditional statements and loops to determine if a year is a leap year."
        ]
      },
      "pljo": { "title": "156", "intro": [] },
```

Reply:
Yeah, that is a good catch. 

This branch might not have the labs that were merged in a few days ago.
@Ksound22  I would suggest merging in the latest changes so you have the recent labs and then updating the order. Remember you can always find the order on the sheet. In this case, the order would be in the functions module which starts are line 425 on the sheet

Reply:
Should be good now

Reply:
You're still removing an existing lab. Also the current `intro.json` from main has a missing placeholder, so you should add:
```
"workshop-loan-qualification-checker": {
        "title": "Build a Loan Qualification Checker",
        "intro": [
          "In this workshop, you will continue to learn how to work with conditionals by building a loan qualification checker app.",
          "You will learn more about <code>if</code> statements, and how to use comparison operators and multiple conditions in an <code>if</code> statement."
        ]
      },
```
in between `lab-sentence-maker` and `lab-leap-year-calculator` without removing any placeholder or existing lab. The order reported in the sheet is:
- 151: calculator app
- 152: email masker
- 153: sentence maker
- 154: loan calculator
- 155: leap year calculator

Reply:
How's it now?

Reply:
It should be as the example in my previous comment. If you want I can push a commit to your branch @Ksound22

Reply:
There's no problem with that.